server/src/Api/Controllers/ApplicationUserController.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Mvc;
+using StudyZen.Api.Extensions;
+using StudyZen.Application.Dtos;
+using StudyZen.Application.Services;
+
+namespace StudyZen.Api.Controllers;
+
+[ApiController]
+public sealed class ApplicationUserController : ControllerBase
```

Comment:
```suggestion
public sealed class AccountController : ControllerBase
```

Maybe we should call it `AccountController`? I think it would make more sense since it is associated with a single (current) user, not all of the application users.